### PR TITLE
libsecret: update to 0.20.5.

### PR DESCRIPTION
**Pull request by tibequadorian**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/libsecret/template [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-configure_args="--enable-introspection --enable-vala --disable-static
- --with-libgcrypt-prefix=${XBPS_CROSS_BASE}/usr"
-hostmakedepends="automake glib-devel gtk-doc pkg-config libxslt vala"
+hostmakedepends="glib-devel pkg-config libxslt vala gi-docgen docbook-xsl
+ gettext"
 makedepends="glib-devel libgcrypt-devel vala"
 short_desc="GObject based library for accessing the Secret Service API"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="http://www.gnome.org"
```
Comment (paper42) [inline srcpkgs/libsecret/template:13]: I think we can figure out a better homepage, maybe `https://wiki.gnome.org/Projects/Libsecret`?

File: srcpkgs/libsecret/template [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
+changelog="https://gitlab.gnome.org/GNOME/libsecret/-/raw/master/NEWS"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=325a4c54db320c406711bf2b55e5cb5b6c29823426aa82596a907595abb39d28
-
-# doesn't work in build chroot
-do_check() {
-	:
-}
+checksum=3fb3ce340fcd7db54d87c893e69bfc2b1f6e4d4b279065ffe66dac9f0fd12b4d
+make_check=no # doesn't work in build chroot
```
Comment (paper42) [inline srcpkgs/libsecret/template:17]:
```suggestion
checkdepends="python3-dbus python3-gobject dbus"
do_check() {
       dbus-run-session ninja -C build test
}
```

File: srcpkgs/libsecret/template (hunk 1 above)
Comment (tibequadorian) [reply srcpkgs/libsecret/template]: that's good

File: srcpkgs/libsecret/template (hunk 2 above)
Comment (tibequadorian) [reply srcpkgs/libsecret/template]: nice

**Pull request merged**